spec/features/user_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+feature "User index" do
+
+  let(:alice) { create(:user, :alice) }
+  let(:bob)   { create(:user, :bob) }
+
+  before do
+    alice.save
+    bob.save
```

Comment:
Yeah, so, before blocks are annoying when you try and refactor your tests.  So we try and avoid them when possible.  Or at least make them configurable.  If ! versions aren't your thing...
```
let(:preconditions) { alice && bob }
before do
  preconditions
  login...
end
```
or something similar.  But typically I expect that most of my ```let``` variables are made in time, so it's not really a huge deal.